framework/src/executioners/Transient.C

```diff
@@ -440,6 +447,24 @@ Transient::solveStep(Real input_dt)
 
       _solution_change_norm = _problem.solutionChangeNorm();
 
+      // Check for stop condition based upon steady-state check flag:
+      if (!_xfem_repeat_step &&
+          _trans_ss_check == true && _time > _ss_tmin)
+      {
+        // Compute new time solution l2_norm
+        NonlinearSystem & nl = _problem.getNonlinearSystem();
+        NumericVector<Number> & current_solution  = (*nl.sys().current_local_solution);
+        NumericVector<Number> & old_solution = (*nl.sys().old_local_solution);
+        NumericVector<Number> & difference = *NumericVector<Number>::build(_communicator).release();
```

Comment:
You'll have to deprecate the existing one... quite a few people use it.

If I remember correctly there is other "naming nonsense" involved with those parameters... let's try to get them *right* this time since we're changing them...